src/alert-manager/src/cert-expiration-checker/send_alert.py

```diff
@@ -0,0 +1,60 @@
+from datetime import timezone, datetime, timedelta
+import logging
+import os
+import requests
+
+ALERT_PREFIX = "/alert-manager/api/v1/alerts"
+alertResidualDays = int(os.environ.get('ALERT_RESIDUAL_DAYS'))
+
+def enable_request_debug_log(func):
+    def wrapper(*args, **kwargs):
+        requests_log = logging.getLogger("urllib3")
+        level = requests_log.level
+        requests_log.setLevel(logging.DEBUG)
+        requests_log.propagate = True
+
+        try:
+            return func(*args, **kwargs)
+        finally:
+            requests_log.setLevel(level)
+            requests_log.propagate = False
+
+    return wrapper
+
+@enable_request_debug_log
+def send_alert(pai_url: str, certExpirationInfo: str):
+    trigger_time = str(datetime.now(timezone.utc).date())
+    post_url = pai_url.rstrip("/") + ALERT_PREFIX
+    alerts = []
+    alert = {
+        "labels": {
+            "alertname": "k8s cert expiration",
+            "severity": "warn",
+            "trigger_time": trigger_time,
+            "annotations": certExpirationInfo,
```

Comment:
Please provide a detailed summary in the `annotations` field like:
```yaml
annoations:
    summary: the k8s cert will be expired in {} days
``` 
The summary will be rendered in the email body

Reply:
Done.